1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,5 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+  
+display(pickFromArray(quotes));
```

Comment:
Missing functionality. Remember we spoke about making a function to do a simple job of display a quote (the quote text and author.

```
// Takes a quote object and displays on the page
function display(quote){
  document.getElementById("author").innerText = quote.author;
  document.getElementById("quote").innerText = quote.quote;
}
```
You've added a button but give the button an event handler (see comment in html). Remember make small functions and build up.  

```
// Event handler for the button click
function showRandomQuote(){
  // Gets a randon quote from the
  // array of quotes and then calls the display
  // function with that quote.
  display(pickFromArray(quotes));
}
```